mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -127,6 +127,63 @@ def parse_split_oriented_json_input_to_numpy(json_input):
         )
 
 
+def parse_tf_serving_input(inp_dict):
+    """
+    :param inp_dict: A dict deserialized from a JSON string formatted as described in TF's
+                     serving API doc
+                     (https://www.tensorflow.org/tfx/serving/api_rest#request_format_2)
+    """
+    # pylint: disable=broad-except
+    if "instances" in inp_dict and "inputs" in inp_dict:
```

Comment:
I think we should check that the input contains exactly one of ("instances", "inputs") and nothing else. In particular, we should look for signature_name and return an error informing users that signature_name is not supported.